projects/storefrontlib/layout/a11y/keyboard-focus/visible/visible-focus.directive.ts

```diff
@@ -22,6 +29,14 @@ import { VisibleFocusConfig } from '../keyboard-focus.model';
  */
 @Directive() // selector: '[cxVisibleFocus]'
 export class VisibleFocusDirective extends BaseFocusDirective {
+  constructor(
+    protected elementRef: ElementRef<HTMLElement>,
+    protected service: BaseFocusService
+  ) {
+    super(elementRef, service);
```

Comment:
I think this call to super could be a breaking change.

Reply:
This should be fine since we only need this to enable the feature flag. The class behaviour stays the same, and we do not intend on keeping those changes. I have added a comment to make sure the constructor gets removed. :)